components/mdc/TextInput/helpers.js

```diff
@@ -1,3 +1,9 @@
 export const addOrRemoveInvalidClass = async (isInvalid, element) => {
   isInvalid ? element.classList?.add('mdc-text-field--invalid') : element.classList?.remove('mdc-text-field--invalid')
 }
+
+export function getDecimalPlacesLength(number = 0) {
+  const string = String(number)
+  const length = string.includes('e-') ? string.split('e-')?.[1] : string.split('.')[1]?.length
+  return length || 0
```

Comment:
You may want to ensure the result is a Number, since the `string.split('e-')?.[1]` bit above returns a numeric string:
```suggestion
  return Number(length || 0)
```